Wage_Map/migrations/0001_initial.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.0.5 on 2022-07-17 14:18
```

Comment:
Is this the script that is generated when you run migrations after defining the model in "Wage_maps" models.py?

Reply:
Yes that's right. This file was auto-generated.